src/web/src/pages/myfeeds.tsx

```diff
@@ -14,9 +13,7 @@ const MyFeedsPage = () => {
 
   return (
     <>
-      <Head>
-        <title>My Feeds</title>
-      </Head>
+      <SEO pageTitle="My Feeds | Telescope" />
```

Comment:
Wouldn't this always result in My Feeds being printed instead of Telescope and thus, not worth keeping?

Reply:
@HyperTHD our current setup actually overwrites `Telescope` on each page, if you think about it in this way. This is page-specific, so for each page, we will have a page title (`My Feeds`, etc) + `| Telescope`. Otherwise when you share specific pages, for example, their titles will just say `About Us`, etc., which is not very explicit. A lot of websites use this format.

Reply:
Ah, makes sense. Thanks for the clarifications!